### ISSUE TITLE
Not an issue; Just a PSA

### ISSUE THREAD

**Issue by Markie98** [CookiePLMonster/SilentPatchBully#54]
The in-game V-Sync must be enabled to achieve proper frame-pacing while utilizing the SilentPatch fps cap, otherwise intermittent micro-stutter may occur every few seconds.

Maybe it would be good to add this information to the description in Silent’s Blog Bully page and the github download page?

**Comment by TheRealDadbeard** [comment]
I don't even think the game runs without vsync and in a proper fullscreen more like borderless. The stuttering that I get on a 60hz display is more like the double buffering from DWM.

I would love if I could get into a real fullscreen. If you have gsync then you'll be fine even at 60fps without input lag. Vsync at 60hz is not a great experience but this game has some big issues without it.

Edit: Yeah game runs in borderless and sadly crashes when forced to fullscreen with specialK

**Comment by Markie98** [comment]
Here it runs in exclusive fullscreen with or without vsync just fine. Actually most games from that era run in exclusive fullscreen and don't even have a borderless mode, so i'm confused about how you're getting it bordeless at all to begin with.

**Comment by TheRealDadbeard** [comment]
Yeah I don't know why too. Just checked with SpecialK and it starts in borderless. Do you have tearing while game runs at 60fps? If not then you're on borderless too.

Edit: I can get the game to run in dx11 with dgvoodoo but then specialK won't hook so I can't force flip mode...

**Comment by CookiePLMonster** [comment]
When debugging I'd occasionally observe the game be both fullscreen and
borderless (the latter being very useful for debugging, actually) - so that
behaviour most likely depends on something I haven't been able to pinpoint
so far.

At some point I even had the game switch to 1080i/50Hz instead of
1080p/60Hz so there's definitely something wrong with resolution handling
in this game.

**Comment by Markie98** [comment]
@TheRealDadbeard I don't remember having tearing in any combination i've tested (default 30fps or 60fps through the silentpatch cap with RTSS/V-Sync on top of it). Though as i said only V-Sync on top of the silentpatch cap eliminated micro-stutter.

@CookiePLMonster That's crazy. 1080i/50hz on a PC game? How would that even happen?
